packages/vetrina/src/Vetrina/Purchase/NewPurchase.purs

```diff
@@ -174,8 +174,9 @@ title self =
   in foldMap headline headlineText
   where
     headline child =
-      DOM.h1
-        { className: "vetrina--headline-" <> maybe "KSF" Paper.toString self.props.paper <>
+      DOM.div
```

Comment:
Just verifying: The change from h1 to div is intentional?

Reply:
I would say so, because in mosaico we add a div element inside this element, so I thought that a div instead of a h1 would be more fitting